doc/pairs.md

```diff
@@ -268,4 +287,17 @@ new_tuples
     - Mikhail
     - 51
 ...
```

Comment:
Add smth like `** THIS WOULD FAIL**` here. We need to highlight it as much as it's possible.